src/Deployer.php

```diff
@@ -134,14 +140,14 @@ public function getOutput()
     /**
      * @param string $name
      * @return mixed
-     * @throws \InvalidArgumentException
+     * @throws InvalidArgumentException
```

Comment:
I disagree. FQCNs should be used if class is not part of file's namespace IMO.
So here it would be `\InvalidArgumentException`